asapdiscovery-docking/asapdiscovery/docking/scripts/run_docking_oe.py

```diff
@@ -124,7 +127,7 @@ def load_dus(file_base, by_compound=False):
     return dataset_dict, du_dict
 
 
-def mp_func(out_dir, lig_name, du_name, *args, **kwargs):
+def mp_func(out_dir, lig_name, du_name, GAT_model, *args, **kwargs):
```

Comment:
```suggestion
def mp_func(out_dir, lig_name, du_name, *args, GAT_model=None, **kwargs):
```
not sure if this is actually necessary, but just to make it clear that the `GAT_model` isn't necessary